.github/actions/sync-to-drive-action/syncgdrive.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
```

Comment:
Optional but would be appreciated: use typing annotations and verify with mypy, e.g. like I do in this unfinished script here: https://github.com/beyond-all-reason/maps-metadata/pull/59

Typing annotations greatly improve readability and catch easy errors.